axelor-account/src/main/java/com/axelor/apps/account/service/AccountManagementServiceAccountImpl.java

```diff
@@ -222,4 +223,16 @@ protected FixedAssetCategory getProductFixedAssetCategory(
 
     return fixedAssetCategory;
   }
+
+  @Override
+  public Account getCashAccount(AccountManagement accountManagement, PaymentMode paymentMode)
+      throws AxelorException {
+    if (accountManagement != null && accountManagement.getCashAccount() == null) {
```

Comment:
If accountManagement is null, we get an NPE, please add a if condition before to check if accountManagement is null and return null. And also in this new if condition, check if paymentMode is null.